jemallocator/README.md

```diff
@@ -65,7 +65,7 @@ other targets are only tested on Rust nightly.
 | `powerpc64le-unknown-linux-gnu`     | ✓         | ✓       | ✗            |
 | `x86_64-unknown-linux-gnu` (tier 1) | ✓         | ✓       | ✓            |
 | **MacOSX targets:**                 | **build** | **run** | **jemalloc** |
-| `x86_64-apple-darwin` (tier 1)      | ✓         | ✓       | ✗            |
+| `aarch64-apple-darwin` (tier 1)      | ✓         | ✓       | ✗            |
```

Comment:
any comment if change `aarch64-apple-darwin` to tier 1 as well?

Reply:
Actually both aarch64-apple-darwin and x86_64-apple-darwin are not tier 1 anymore. I suggest to remove the tier 1 statement instead.